app/src/main/java/com/teamltt/carcare/database/DbHelper.java

```diff
@@ -238,6 +314,81 @@ public int updateVehicle(long vehicleId, Vehicle vehicle) {
         return numAffected;
     }
 
+    public long createNewReminder(long vehicleId, String name, int featureId, int comparison, int value, String date)  {
+        SQLiteDatabase db = getWritableDatabase();
+        long status = DbHelper.errorChecks(db);
+        if (status != DbHelper.DB_OK) {
+            return status;
+        }
+        status = ReminderContract.insert(db, vehicleId, name, featureId, comparison, value, date);
+        db.close();
+        setChanged(status);
+        return status;
+    }
+
+    public long updateReminder(int reminderId, long vehicleId, String name, int featureId, int comparison, int value, String date) {
```

Comment:
Same. The difference would be that this method also takes in the `long reminderId`.